org.gnome.gThumb.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.gnome.gThumb
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+
+command: gthumb
+rename-icon: gthumb
```

Comment:
Can you add a `copy-icon: true` here? Otherwise the icon in the AboutDialog is missing
![Screenshot from 2019-12-19 12-02-48](https://user-images.githubusercontent.com/7660997/71168725-a42af800-2257-11ea-8648-15388241800b.png)
If possible, this should be reported/patched upstream

Reply:
I'm happy to file a bug, but I don't fully understand what the issue is that upstream has to change.
Do you want upstream to ship the icon as `org.gnome.gThumb.png` rather than `gthumb.png` and make it use that icon the about dialogue?

Reply:
yeah, basically either we need to copy-icon or upstream shifts to the 'flatpak style' naming for all exportable assets

Reply:
https://gitlab.gnome.org/GNOME/gthumb/issues/77